sdk/core/azure-core/src/main/java/com/azure/core/annotation/FormParam.java

```diff
@@ -31,7 +31,7 @@
     /**
      * A value true for this argument indicates that value of {@link FormParam#value()} is already encoded
      * hence engine should not encode it, by default value will be encoded.
```

Comment:
I find the differences in wording between here and `HostParam` / `PathParam` / `QueryParam` a little confusing, about default encoding state and behaviours. Can you expand these sections to be more clear?